routes/quiz.js

```diff
@@ -6,11 +6,15 @@ const {
 const quizController = require('~/controllers/quiz')
 const asyncWrapper = require('~/middlewares/asyncWrapper')
 const { authMiddleware, restrictTo } = require('~/middlewares/auth')
+const isEntityValid = require('~/middlewares/entityValidation')
+const Quiz = require('~/models/quiz')
+
+const params = [{ model: Quiz, idName: 'id' }]
 
 router.use(authMiddleware)
 
 router.use(restrictTo(TUTOR))
-router.get('/', asyncWrapper(quizController.getQuizzes))
-router.post('/', asyncWrapper(quizController.createQuiz))
+
+router.delete('/:id', isEntityValid({ params }), asyncWrapper(quizController.deleteQuiz))
 
```

Comment:
Why did you delete get and post endpoints?

Reply:
I've done it accidentally. Fixed.